email_alert_notifications/jenkins.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -e
+export FILE_TO_UPLOAD=rename_email_files_with_request_id.py.zip
+mkdir -p archive/$(date +%s)
+mv $FILE_TO_UPLOAD archive/$(date +$s)/
+/usr/bin/zip $FILE_TO_UPLOAD rename_email_files_with_request_id.py
```

Comment:
It's meant to be archiving anything that might already exist in there, though the syntax is completely wrong, should probably be `*.zip` rather than `$FILE_TO_UPLOAD` on line 6. But actually, if the workspace is wiped after every run then this shouldn't be required at all.